EIPS/eip-7400.md

```diff
@@ -0,0 +1,211 @@
+---
+eip: 7400
```

Comment:
```suggestion
eip: 7399
```
Manually assigning first PR number #7399.  Number sniping is not allowed.  PR 7400 was made one minute after a one line change PR 7399 by the same author.

Please also update the filename.

Reply:
Please use the assigned number 7399 for the EIP.  

Given the PRs could have been done in any order, to avoid any perception of number sniping, the first PR number should be used.